Mathlib/Algebra/Module/LinearMap/Defs.lean

```diff
@@ -981,3 +988,48 @@ def restrictScalarsₗ : (M →ₗ[S] N) →ₗ[R₁] M →ₗ[R] N where
 end RestrictScalarsAsLinearMap
 
 end LinearMap
+
+namespace IsLinearMap
+
+variable [Semiring R] [AddCommMonoid M] [AddCommMonoid M₂] [AddCommMonoid M₃]
+variable [Module R M] [Module R M₂] [Module R M₃]
+variable {ι : Type*}
+
+variable (R) in
+abbrev mk'' (f : M → M₂) (lin : IsLinearMap R f) : M →ₗ[R] M₂ := mk' f lin
```

Comment:
Isn't this the same as `mk'`? Why do you need both?

Reply:
The only difference is that `mk''` has `R` as explicit parameter. The only reason for this function existing is that I do not know how to write an unexpander if I use `mk'`

Reply:
My guess would be that an unexpander no longer cuts it and instead you need a delaborator.

Reply:
Yes, for `mk'` you need delaborator but I got lost in writing it.

Reply:
I dropped `mk''` and wrote the delaborator